CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing 
+
+Everyone is welcome to contribute to this specification.
+
+Any simple editorial contribution can simply be done with a GitHub Pull Request.
+You can even do an inline edit of the file on GitHub.
+
+For more substantial contributions, please first start a thread in the
```

Comment:
Let's delete this as no one ever does this. We need to handle this in GitHub issues. The IPR bot will a catch any non-members trying to add things to the spec.